src/Struct/Marketplace/Product.php

```diff
@@ -52,19 +58,24 @@ public function getId(): string
         return $this->id;
     }
 
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
     /**
-     * @return int
+     * @return string
      */
-    public function getPrice(): int
+    public function getText(): string
```

Comment:
Maybe, we should add a docblock to describe the content - it's a json-dataset, right?

Reply:
nope it is a plain text string now as we discussed several month ago